modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -239,18 +399,91 @@ public void testGetObjectEntry() throws Exception {
 			title, MapUtil.getString(objectEntry.getProperties(), "title"));
 	}
 
+	private void _assertEquals(
+			List<ObjectEntry> expectedObjectEntries,
+			List<ObjectEntry> actualObjectEntries)
```

Comment:
Remember to sort alphabetically
```suggestion
			List<ObjectEntry> actualObjectEntries,
			List<ObjectEntry> expectedObjectEntries
```